src/main/kotlin/traffic_simulation/Main.kt

```diff
@@ -1,29 +1,76 @@
 package traffic_simulation
 
+// Imports for parsing library
+import com.univocity.parsers.common.record.Record
+import com.univocity.parsers.csv.CsvParser
+import com.univocity.parsers.csv.CsvParserSettings
+import java.io.*
+
 fun main(args: Array<String>) {
 
-    scenario(testList, testRoad)
+    val testRoad : RoadNetwork = RoadNetwork(capacity = 8.0)
+
+    val driveInterest : MutableList<Vehicle> = parseInputOfCSV(fileName = "driveInterest.csv")
```

Comment:
Du erinnerst dich vielleicht, dass man in IntelliJ Ordner gelegentlich als _Source Root_ o.ä. markieren musste. Das liegt daran, dass die IDE (das gilt für Maven aber genau so) wissen muss, wo sie nach Quellcode, Ressourcen, Konfigurationen, ... suchen muss. Damit man nicht immer jedes Detail einstellen muss, gilt meist "Convention over Configuration", d.h. diese Tools machen gewissen Annahmen wo sie manche Sachen finden, und wenn man davon nicht abweicht, muss man euch nichts konfigurieren.

* `src/main/resources` ist ein Beispiel, in dem wir uns an die Konvention halten wollte. wir haben nichts konfiguriert, dann aber die Datei falsch untergebracht.
* `src/main/kotlin` ist ein Beispiel, in dem wir von der Konvention abweichen und deswegen Maven entsprechend konfigurieren müssen.

Generell versteht IntelliJ Maven-Konfigurationen, so dass man nicht alles dopeelt einstellen muss.